Cargo.toml

```diff
@@ -77,6 +79,7 @@ p256 = { version = "0.8", optional = true, features = ["zeroize", "ecdsa"] }
 ssi-contexts = { version = "0.1.2", path = "contexts/" }
 ripemd160 = { version = "0.9", optional = true }
 sshkeys = "0.3"
+sequoia-openpgp = "1.7"
```

Comment:
Since the changes are now all in the did-webkey crate, it looks like this dependency isn't needed here.
It might be needed again though when implementing signing/verification.

Reply:
Sure thing, I'll remove that.